subsys/bluetooth/controller/ll_sw/ull_central_iso.c

```diff
@@ -953,6 +955,28 @@ static void mfy_cis_offset_get(void *param)
 			cig_remainder_us + cig->sync_delay -
 			acl_remainder_us - cis->sync_delay;
 
+	/* Calculate instant latency */
+	latency_acl = cis->central.instant - ull_conn_event_counter(conn);
+	elapsed_acl_us = latency_acl * conn->lll.interval * CONN_INT_UNIT_US;
+
+	/* Calculate elapsed CIG intervals until the instant */
+	cig_interval_us = cig->iso_interval * ISO_INT_UNIT_US;
+	latency_cig = DIV_ROUND_UP(elapsed_acl_us, cig_interval_us);
+	elapsed_cig_us = latency_cig * cig_interval_us;
+
+	/* Compensate for the difference between ACL elapsed vs CIG elapsed */
+	offset_min_us += elapsed_cig_us - elapsed_acl_us;
+	while (offset_min_us > (cig_interval_us + 500U)) {
+		offset_min_us -= cig_interval_us;
+	}
```

Comment:
Where does the `500U` come from?

Reply:
This is the spec defined minimum value for CIS offset, i did not find another use of this value in the implementation. On second thought, I will have a define for this.

Reply:
Added define.